src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
```

Comment:
missing test cases:
* existing user with the same login(keep in mind the way how we should properly add the first user, check common mistakes for it)
* null user itself